static/app/gettingStartedDocs/python/aiohttp.tsx

```diff
@@ -127,56 +139,29 @@ app.add_routes([web.get('/', hello)])
 web.run_app(app)
 `,
         },
-        ...(params.isLogsSelected
-          ? [
-              {
-                description: t(
-                  'You can send logs to Sentry using the Sentry logging APIs:'
-                ),
-                language: 'python',
-                code: `import sentry_sdk
-
-# Send logs directly to Sentry
-sentry_sdk.logger.info('This is an info log message')
-sentry_sdk.logger.warning('This is a warning message')
-sentry_sdk.logger.error('This is an error message')`,
-              },
-              {
-                description: t(
-                  "You can also use Python's built-in logging module, which will automatically forward logs to Sentry:"
-                ),
-                language: 'python',
-                code: `import logging
-
-# Your existing logging setup
-logger = logging.getLogger(__name__)
-
-# These logs will be automatically sent to Sentry
-logger.info('This will be sent to Sentry')
-logger.warning('User login failed')
-logger.error('Something went wrong')`,
-              },
-            ]
-          : []),
+        getVerifyLogsContent(params),
+        {
+          type: 'custom',
+          content: (
+            <span>
+              <p>
+                {tct(
+                  `When you point your browser to [localhostLInk:http://localhost:8080/] a transaction in the Performance section of Sentry will be created.`,
+                  {
+                    localhostLInk: <ExternalLink href="http://localhost:8080/" />,
+                  }
+                )}
+              </p>
+              <p>
+                {t(
+                  'Additionally, an error event will be sent to Sentry and will be connected to the transaction.'
+                )}
+              </p>
+              <p>{t('It takes a couple of moments for the data to appear in Sentry.')}</p>
+            </span>
+          ),
+        },
```

Comment:
I would not use custom but instead 2x text blocks, so we can have a bit of space between these two paragraphs:

<img width="854" height="77" alt="image" src="https://github.com/user-attachments/assets/2129ae9c-0de1-47de-abf1-628b850e40a4" />